dockstore-webservice/src/main/java/io/dockstore/webservice/core/gitlab/package-info.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2019 OICR
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Models for responses from GitLab. Used right now just to grab checksum info for images. But if other fields are needed
```

Comment:
Explain how the classes in this package were created and how one would update it (same with Docker Hub really)
(i.e. I'm guessing this was manually constructed by looking at the responses from gitlab, but maybe there was documentation somewhere or a swagger-ui equivalent)